CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+## Setting up a dev environment
+
+Though voila-gallery is deployed as a [TLJH](https://tljh.jupyter.org)
+plugin, you can develop it locally without needing TLJH. You just need
```

Comment:
Then TLJH is still required? (at least as a dev dependency)

Reply:
Yep it is, mostly because we're importing it unconditionally in a bunch of places.

Reply:
It won't do anything to your system though... And we *could* remove it if needed. I'm ok with leaving it in though.

Reply:
ok we can keep it. Then we can remove this part: "you can develop it locally without needing TLJH".

Reply:
I clarified it to mean you don't have to *install* TLJH, which takes forever and shouldn't be done on your own system. Is that clearer?

Reply:
yes!

Reply:
Awesome :) <3